packages/contrast-colors/index.js

```diff
@@ -488,27 +785,46 @@ function luminance(r, g, b) {
 }
 
 function contrast(color, base, baseV) {
+  if(baseV == undefined) { // If base is an array and baseV undefined
+    let colorString = String(`rgb(${base[0]}, ${base[1]}, ${base[2]})`);
```

Comment:
```suggestion
    let colorString = `rgb(${base[0]}, ${base[1]}, ${base[2]})`;
```
It's a string by default, not need to try to cast it.